testgen/icu75/message_fmt2/formatter/number.json

```diff
@@ -0,0 +1,264 @@
+{
+  "scenario": "",
+  "description": "",
+  "tests": [
+  ]
+}
```

Comment:
is this file proper JSON? the object that begins online 1 ends on line 6.

beyond that, this file and some others that follow don't specify the default locale, FYI.